decidim-core/app/models/decidim/category.rb

```diff
@@ -20,6 +20,14 @@ def self.first_class
       where(parent_id: nil)
     end
 
+    def descendants
+      Category.where(parent_id: self.id)
```

Comment:
`@descendants ||=`

Reply:
Ok, I will do it.
But I do not know very well what it brings to equal it to `@descendants`, this is used in the `grouped_collection_select` of Proposal Index of the admin https://github.com/decidim/decidim/blob/a3b38366917c8736d8dd57745c03e00427e16563/decidim-proposals/app/views/decidim/proposals/admin/proposals/index.html.erb#L26

Reply:
I mean to memoize it, so if we call this method twice we don't hit the DB again.